src/plugin/reflection_class.cc

```diff
@@ -13,6 +17,21 @@ namespace chapel {
   void ReflectionClassGenerator::Generate(Printer* printer) {
     WriteIntroduction(printer);
 
+    string module_name;
+    if (!package_name.empty()) {
+      module_name = package_name;
+    } else {
+      module_name = default_file_name;
+    }
```

Comment:
It seems like this should use the same pathway as the code in GetOutputFile, so changes don't need to be made to two places in the future

Reply:
Yes, makes sense but from the return  of `GetOutputFile` we have to strip `.chpl` out. So I see two options here->

1) Either modify `GetOutputFile` to just return the base name and we append `.chpl` latter. In this case we can directly use `GetOuputFile` here.

2) Use `GetOutputFIle`here and strip `.chpl`.

Which one do you prefer?

Reply:
What about having GetOutputFile rely on a helper function that both places can call, having GetOutputFile merely append `.chpl` onto the result of that so that it still reasonably fulfills its name and purpose?

Reply:
Yes, sounds great! How about GetOuputModule? GetOutputFile calls it and appends .chpl to the output?

Reply:
That sounds like a good idea to me :)

Reply:
I have the changes in 1263928, please have a look.

Reply:
Looks great, thanks!